docs/topics/modules/changing_quickstart_code.adoc

```diff
@@ -25,7 +25,7 @@ The workspace automatically saves your changes.
 
 . To test your changes:
 
-** If you already ran the application earlier as instructed in <<configuring_your_che_workspace-hello-world>>, your changes are instantly implemented. Maven uses the hot deploy feature provided by Vert.x to automatically update the application when you make a change. Return to the browser tab running the application and skip the rest of this step.
+** If you already ran the application earlier as instructed in <<configuring_your_che_workspace-hello-world>>, your changes are instantly implemented. Maven uses Vert.X's hot deploy feature to automatically update the application when you make a change. Return to the browser tab running the application and skip the rest of this step.
```

Comment:
@mishaone Do you think this change makes the sentence easier to read? I would normally argue against using possessives with inanimate nouns.

Reply:
Agreed, changed in new PR.